Core/DISET/RequestHandler.py

```diff
@@ -114,7 +114,7 @@ def _rh_executeAction(self, proposalTuple):
                         of action to execute. The second position is the action itself.
     """
     actionTuple = proposalTuple[1]
-    gLogger.debug("Executing %s:%s action" % actionTuple)
+    gLogger.debug("Executing %s:%s action" % tuple(actionTuple))
```

Comment:
Out of curiosity, how can that happen ? 
Moreover, the rest will crash as well no  ? `actionType` would be just one caracter for example (2 lines bellow), and then you would get `S_ERROR("Unnown action")`

Reply:
I encountered this problem when making tests with v7r1 prerelease. I guess this is due to your updates for the JSON based protocol: ActionTuple is actually a list rather than a tuple. The rest of the code should still work in this case for both v7r0 and v7r1

Reply:
Ah very true ! I did not think that the string formatting would behave differently ! 
But hold on, that means that you are using v7r1 client with v7r0 server ? That cannot work.
In v7r1, I've added a `_deserialzieProposalTuple` method, which should cast all that back to tuple when receiving, and unless there is a bug, you should not see this issue then

Reply:
Indeed this was v7r1 client accessing v7r0 service. The fix made it working again. I guess the use of the ActionTuple is the same as if it would be a list.

Reply:
But that should not work anyway, since you send an extra parameter in v7r1 client (the client version) that the v7r0 service is not able to understand ?
Anyway, maybe the fixes are better targeting v7r1, if that's where the problem is no ?

Reply:
These are fixes in v7r0 for services to be forward compatible with v7r1 client. This is indeed a rare case but does not harm v7r0 service and can avoid some incompatibilities in the future.
ExtraCredentials is a two element tuple/list both in v7r0 and in v7r1

Reply:
No it does not harm, but this https://github.com/DIRACGrid/DIRAC/pull/4236/ still breaks the compatibility I believe (at least in the tests I've done just before Christmas)

Reply:
I honestly do not see how #4236 breaks compatibility. But should be tested of course